src/Microsoft.FeatureManagement/ServiceCollectionExtensions.cs

```diff
@@ -57,8 +40,16 @@ public static IFeatureManagementBuilder AddFeatureManagement(this IServiceCollec
                 sp.GetRequiredService<IOptions<TargetingEvaluationOptions>>())
             {
                 Configuration = sp.GetService<IConfiguration>(),
-                TargetingContextAccessor = sp.GetService<ITargetingContextAccessor>()
+                TargetingContextAccessor = sp.GetService<ITargetingContextAccessor>(),
+                TelemetryPublishers = sp.GetService<IOptions<FeatureManagementOptions>>()?.Value.TelemetryPublisherFactories?
```

Comment:
Can `TelemetryPublishers` be null? If yes, then I think there might be some missing null checks in `FeatureManager`

Reply:
In that case might also want to add a test with telemetry enabled in the feature, but no publisher